sources/platform/schedules.md

```diff
@@ -6,79 +6,84 @@ category: platform
+In [Apify Console](https://console.apify.com/schedules), click on the **Schedules** in the navigation menu, then click the **Create new** button.
 
-In the **Settings** tab, give your schedule a memorable name, add a description, and choose how often you would like your Actor or task to run using the [schedule setup tool](#schedule-setup).
+Click on the name (by default it is _My Schedule_), there you can change the name of Schedule, add a description, as well as check it's _Unique name_ or _ID_.
 
-![New schedule](./images/schedule-settings.png)
+You can adjust how often your Actor or task runs using the [schedule setup tool](#schedule-setup). You can find it by clicking on **Schedule setup** tab.
+
+![New schedule](./images/schedules-overview.png)
 
 Next, you'll need to give the schedule something to run. This is where the Actor or task you prepared earlier comes in. Switch to the **Actors and Tasks** tab, and click the **Add [new]** button.
 
-If you're scheduling an Actor run, you'll be able to specify the Actor's [input](./actors/running/input_and_output.md) and running options like
-[build](./actors/development/builds_and_runs/builds.md),
-timeout,
-[memory](./actors/running/usage_and_resources.md).
-The **timeout** value is specified in seconds; a value of **0** means there is no timeout, and the Actor runs until it finishes.
+If you're scheduling an Actor run, you'll be able to specify the Actor's [input](./actors/running/input_and_output.md) and running options like [build](./actors/development/builds_and_runs/builds.md), timeout, [memory](./actors/running/usage_and_resources.md).
+The **timeout** value is specified in seconds; a value of _0_ means there is no timeout, and the Actor runs until it finishes.
 
  If you don't provide an input, then the Actor's default input is used. If you provide an input with some fields missing, the missing fields are filled in with values from the default input. If input options are not provided, the default options values are used.
 
-![Add Actor to schedule](./images/schedule-actor-run.png)
+![Add Actor to schedule](./images/schedules-actor-input.png)
 
 If you're scheduling a task, just select the task you prepared earlier using the drop-down. If you need to override the task's input, you can pass it as a JSON object in the **Input JSON overrides** field.
 
-![Add task to schedule](./images/schedule-add-tasks.png)
+![Add task to schedule](./images/schedules-task-input.png)
 
 To add more Actors or tasks, just repeat the process.
 
-> You can add a maximum of 10 Actors and 10 tasks to each schedule.
-
 Now, all you need to do is click **Save & activate** and let the scheduler take care of running your jobs on time.
 
 For integrations, you can also add a [webhook](/platform/integrations/webhooks) to your tasks, which will notify you (or perform an action of your choice) every time the task runs.
 
-### Via API
+### Apify API
```

Comment:
Question: I guess we might have this section, briefly describing how to work with API, in all relevant places in docs. Or maybe just link to some "Getting started with API" section + link to docs of relevant endpoints?